### PR TITLE
fix a few typos

### PR DESCRIPTION
**Pull request by Findus23**
Hi,

I noticed a few spelling errors in the `angular_en_US.xml`

### CODE REVIEW

**Comment by Chocobozzz** [comment]
Hi,

Thanks but it's a generated file. You have to update the HTML instead :)

**Comment by Findus23** [comment]
Ah, that makes a lot of sense.
I'll update it.

File: server/tests/api/check-params/users.ts [hunk 1]
```diff
@@ -901,7 +901,7 @@ describe('Test users API validators', function () {
       await makePostBodyRequest({ url: server.url, path: registrationPath, token: server.accessToken, fields })
     })
 
-    it('Should fail with a channel name that is the same than user username', async function () {
+    it('Should fail with a channel name that is the same as user username', async function () {
```
Comment (BO41) [inline server/tests/api/check-params/users.ts]: is "user username" correct? sounds a bit strange

Comment (Findus23) [reply server/tests/api/check-params/users.ts]: You are totally right, I missed that one.

**Comment by Chocobozzz** [comment]
Thanks!

**Pull request merged**